src/Framework/AspNetCoreAnalyzers/test/Verifiers/CSharpWebApplicationBuilderCodeFixVerifier.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using Microsoft.AspNetCore.Analyzers.WebApplicationBuilder;
+using Microsoft.AspNetCore.Analyzers.WebApplicationBuilder.Fixers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+
+namespace Microsoft.AspNetCore.Analyzers.WebApplicationBuilder;
+
+public static class CSharpWebApplicationBuilderCodeFixVerifier
+{
+    public static DiagnosticResult Diagnostic(string diagnosticId = null)
+        => CSharpCodeFixVerifier<WebApplicationBuilderAnalyzer, WebApplicationBuilderFixer, XUnitVerifier>.Diagnostic(diagnosticId);
+
+    public static DiagnosticResult Diagnostic(DiagnosticDescriptor descriptor)
+        => new DiagnosticResult(descriptor);
+
+    public static Task VerifyAnalyzerAsync(string source, params DiagnosticResult[] expected)
+    {
+        var test = new CSharpWebApplicationBuilderAnalyzerVerifier.Test { TestCode = source };
+        test.ExpectedDiagnostics.AddRange(expected);
+        return test.RunAsync();
```

Comment:
```suggestion
        return VerifyCodeFixAsync(source, expected, source);
```

Then we can delete the whole `CSharpWebApplicationBuilderAnalyzerVerifier`